### PR TITLE
elasticsearch events and alarms forwarder (OSGI feature)

### PR DESCRIPTION
**Pull request by unicolet**
this is a rewrite of my earlier work demoed at devjam

The [wiki page](http://www.opennms.org/wiki/Dev-Jam_2015/elasticsearch) documents configuration and I tracked implemented features with this [trello board](https://trello.com/b/JJsvvlAv/opennms-elasticsearch).

Thanks to everyone at DevJam and tip_of_the_hat to @dschlenk whose work I used as a starting point.

### CODE REVIEW

**Comment by j-white** [comment]
JIRA: http://issues.opennms.org/browse/NMS-7903
Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS186

**Comment by j-white** [comment]
Is the 'features/elasticsearch/event-forwarder/src/main/resources/names.txt' file necessary? It doesn't appear to be referenced anywhere.

File: pom.xml [hunk 1]
```diff
@@ -1219,6 +1219,7 @@
     <activemqVersion>5.10.0</activemqVersion>
     <atomikosVersion>3.9.2</atomikosVersion>
     <camelVersion>2.14.1</camelVersion>
+    <elasticsearch-version>1.0.0_4</elasticsearch-version>
```
Comment (j-white) [inline pom.xml]: This property doesn't appear to be referenced anywhere.

**Comment by unicolet** [comment]
names.txt is required per https://issues.apache.org/jira/browse/CAMEL-8194

File: pom.xml (hunk 1 above)
Comment (unicolet) [inline pom.xml]: Good catch! I probably meant to use it [here](https://github.com/OpenNMS/opennms/pull/336/files#diff-0eb738dd4425240b8e667d41580adf31R76) but missed it

Comment (soleger) [inline pom.xml]: Please update this to 'elasticsearchVersion' so that it matches all other properties also.

Comment (pbrane) [inline pom.xml]: > On Oct 6, 2015, at 2:32 PM, Seth Leger <notifications@github.com> wrote:
> 
> > +    <elasticsearch-version>1.0.0_4</elasticsearch-version>
I like the “-“ in XML for some reason as a personal preference but I gave up fighting that battle long about ;)

Thanks for review the code and working with Umberto, guys!

**Comment by j-white** [comment]
Interesting. Overall it looks good to me. We'll  want to migrate the wiki docs to asciidoc though.

I'm currently in the process of trying it out with ElasticSearch and Kibana.

**Comment by unicolet** [comment]
@j-white how did it go? I soon might be able to set aside a day to get this merged properly

**Comment by j-white** [comment]
Thanks for the update @unicolet. Looks good.

I'll run it through Bamboo and merge it if all goes well.

**Pull request merged**